### DISCUSSION TITLE
Import error

### DISCUSSION THREAD

**Discussion by Pratik-Bhatt190** [corteva/rioxarray#577] [General]
Hi, I am currently using anaconda distribution with python 3.6. I have installed rioxarray from both anaconda and Pip, but while I am 
![Screenshot (112)](https://user-images.githubusercontent.com/103387711/190181649-b2d30ba9-93a3-4a2a-b7f0-a772c057ceeb.png)
trying to import it in my jupyter notebook it's showing errors.

**Comment by snowman2** [comment]
See #451. rioxarray supports Python 3.8+.
